src/main/java/apoc/export/graphml/XmlGraphMLWriter.java

```diff
@@ -97,7 +103,7 @@ private void writeLabels(XMLStreamWriter writer, Node node) throws IOException,
     private void writeLabelsAsData(XMLStreamWriter writer, Node node) throws IOException, XMLStreamException {
         String labelsString = getLabelsString(node);
         if (labelsString.isEmpty()) return;
-        writeData(writer, "labels", labelsString);
+        writeData(writer, "label", labelsString);
```

Comment:
this changes the format (backwards compat) (perhaps better to write both, or add a config option for the old format)

are we sure that what Gephi calls label is the same as our labels?
I thought it was rather something like a caption.

Reply:
This change is mandatory in order to correctly display labels in Gephi, maybe we can add, as you suggested, a property like `{exportFormat:'Gephi'}`, what do you think?

Reply:
perhaps just "format"

Reply:
as of our conversation, "label" in gephi is rather a caption and what we export here seems to be "TYPE", see apoc.export.gephi procedure implementation.